include/wx/webview.h

```diff
@@ -223,6 +223,10 @@ class WXDLLIMPEXP_WEBVIEW wxWebView : public wxControl
 protected:
     virtual void DoSetPage(const wxString& html, const wxString& baseUrl) = 0;
 
+    //Counts number of calls of RunScript, to avoid
```

Comment:
I am sorry, I must be missing something basic here as I cannot understand the purpose of `m_runScriptCount`. How can the method-local `wxJSScriptWrapper` instance be used more than once? This made sense only in the original code which was just a function with nowhere to store the per `RunScript()` "cookie" variable... If the variable must be used, cannot the counter be somehow contained inside `wxJSScriptWrapper`...